tests/x11/yast2_lan_restart.pm

```diff
@@ -0,0 +1,311 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use strict;
+use testapi;
+
+#https://progress.opensuse.org/issues/11450
+
+sub run_yast2_lan {
+    type_string "yast2 lan\n";
+    assert_screen 'yast2_lan', 100;    # yast2 lan overview tab
+}
+
+sub run_yast2_lan_edit {
+    run_yast2_lan;
+    send_key 'alt-i';                  # Edit NIC
+    assert_screen 'yast2_lan_network_card_setup';
+}
+
+sub close_yast {
+    wait_still_screen;
+    send_key 'alt-o';                  # OK
+    if (check_screen 'yast2_lan_packages_need_to_be_installed', 5) {
+        send_key 'alt-i';              # Install
+    }
+    assert_screen 'yast2_closed_xterm_visible', 120;
+    script_run 'ip a';
+    script_run 'cat /dev/null > strace.log';    # clear strace.log
+}
+
+sub check_network {
+    my $status = shift || 'no_restart';
+    wait_still_screen;
+    send_key 'alt-o';                           # OK
+    assert_screen 'yast2_closed_xterm_visible', 120;
+    script_run 'ip a';
+    # workaround bsc#992113
+    #assert_script_run 'dig suse.com|grep status';    # test if conection and DNS is working
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit